framework/web/router.go

```diff
@@ -57,12 +56,12 @@ const (
 func (r *Router) Inject(
 	cfg *struct {
 		// base url configuration
-		Scheme       string         `inject:"config:flamingo.router.scheme,optional"`
-		Host         string         `inject:"config:flamingo.router.host,optional"`
-		Path         string         `inject:"config:flamingo.router.path,optional"`
-		External     string         `inject:"config:flamingo.router.external,optional"`
-		SessionStore sessions.Store `inject:",optional"`
-		SessionName  string         `inject:"config:flamingo.session.name,optional"`
+		Scheme       string        `inject:"config:flamingo.router.scheme,optional"`
+		Host         string        `inject:"config:flamingo.router.host,optional"`
+		Path         string        `inject:"config:flamingo.router.path,optional"`
+		External     string        `inject:"config:flamingo.router.external,optional"`
+		SessionStore *SessionStore `inject:""`
```

Comment:
This is backwards incompatible, but the router should not be used for unittests anyway...

Reply:
The we could also move it out of the cfg since it is not optional anymore

Reply:
This did not resolve the point I had. The `Inject` signature changed, which makes it backwards incompatible